build/karma.conf.babel.js

```diff
@@ -42,23 +42,19 @@ module.exports = (karmaConfig) => {
     },
     webpack: {
       devtool: 'cheap-module-source-map',
-      module: {
-        ...webpackConfig.module,
+      module: Object.assign({}, webpackConfig.module, {
```

Comment:
I know, but riddle me this:

**spread.js**
```js
const objA = {
  foo: 'bar'
}

const objB = { ...objA }
```
```sh
$ node -v
v6.2.1

$ node spread.js                                                                                                                                                                                     [node 6.2.1/3.9.3] [rb 2.3.1] [py sys] 
/Users/levithomason/src/stardust/spread.js:5
const objB = { ...objA }
               ^^^
SyntaxError: Unexpected token ...
    at Object.exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:513:28)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
    at Function.Module.runMain (module.js:575:10)
    at startup (node.js:160:18)
    at node.js:456:3
```

There must be some flag or some other config required.  I just don't want to risk it not running for anyone when `Object.assign()` is available.